src/ReachSets/ContinuousPost/TMJets/post.jl

```diff
@@ -1,6 +1,68 @@
 using TaylorSeries: set_variables
 using LazySets.Approximations: box_approximation
 
+import TaylorModels
+import IntervalArithmetic
+const IA = IntervalArithmetic
+const zeroI = IA.Interval(0.0) # [0.0, 0.0]
+const oneI = IA.Interval(1.0) # [1.0, 1.0]
+const symI = IA.Interval(-1.0, 1.0)
+@inline zeroBox(m) = IntervalBox(zeroI, m)
+@inline symBox(m) = IntervalBox(symI, m)
+
+function _to_hyperrectangle(tTM, xTM, n)
+    N = length(xTM)
+    SET_TYPE = Hyperrectangle{Float64, SVector{n, Float64}, SVector{n, Float64}}
+    Rsets = Vector{ReachSet{SET_TYPE}}(undef, N-1)
+    @inbounds for i in 1:N-1
+        Hi = convert(Hyperrectangle, xTM[i])
+        t0 = tTM[i]; t1 = tTM[i+1]
+        Rsets[i] = ReachSet(Hi, t0, t1)
+    end
+    return Rsets
+end
+
+function _to_intervalbox(tTM, xTM, n)
+    N = length(xTM)
+    SET_TYPE = IA.IntervalBox{n, Float64}
+    Rsets = Vector{ReachSet{SET_TYPE}}(undef, N-1)
+    @inbounds for i in 1:N-1
+        Bi = xTM[i]
+        t0 = tTM[i]; t1 = tTM[i+1]
+        Rsets[i] = ReachSet(Bi, t0, t1)
+    end
+    return Rsets
+end
+
+function _to_zonotope(tTM, vTM, n)
+    N = length(tTM)
+    SET_TYPE = Zonotope{Float64}
+    Rsets = Vector{ReachSet{SET_TYPE}}(undef, N-1)
+    @inbounds for i in 1:N-1
+        # pick the i-th Taylor model
+        X = vTM[:, i]
+
+        # pick the time domain of the given TM (same in all dimensions)
+        Δt = TaylorModels.domain(X[1])
+
+        # evaluate the Taylor model in time, the coefficents are now intervals
+        X_Δt = evaluate(X, Δt)
+
+        # builds the associated taylor model for each coordinate
+        X̂ = [TaylorModelN(X_Δt[k], X[k].rem, zeroBox(n), symBox(n)) for k in 1:n]
```

Comment:
Not sure, but would it make sense to share the `zeroBox(n)` and `symBox(n)`?
```julia
zero_box = zeroBox(n)
sym_box = symBox(n)
X̂ = [TaylorModelN(X_Δt[k], X[k].rem, zero_box, sym_box) for k in 1:n]
```
I would also use `i` as iterator for `1:n`.

Reply:
> I would also use i as iterator for 1:n.

We changed it to "j". Note that "i" is used to refer to the index of the reachset.

Reply:
> Not sure, but would it make sense to share the zeroBox(n) and symBox(n)?

These are the zeros and remainders for *different* Taylor models; i dont think they should be shared.

Reply:
> We changed it to "j". Note that "i" is used to refer to the index of the reachset.

In other places we used `k` for the reachset indices.